azext_iot/_params.py

```diff
@@ -378,3 +385,66 @@ def load_arguments(self, _):
 
     with self.argument_context('iot dps registration list') as context:
         context.argument('enrollment_id', help='ID of enrollment group')
+
+    with self.argument_context('iot dt') as context:
+        context.argument('repo_login', options_list=['--repo-login', '--rl'],
+                         help='This command supports an entity connection string with rights to perform action. '
+                         'Use to avoid PnP endpoint and repository name if repository is private. '
+                         'If both an entity connection string and name are provided the connection string takes priority.')
+        context.argument('interface', options_list=['--interface', '-i'],
+                         help='Target interface name. This should be the name of the interface not the urn-id.')
+        context.argument('command_name', options_list=['--command-name', '--cn'],
+                         help='IoT Plug and Play interface command name.')
+        context.argument('command_payload', options_list=['--command-payload', '--cp', '--cv'],
+                         help='IoT Plug and Play interface command payload. '
+                         'Content can be directly input or extracted from a file path.')
+        context.argument('interface_payload', options_list=['--interface-payload', '--ip', '--iv'],
+                         help='IoT Plug and Play interface payload. '
+                         'Content can be directly input or extracted from a file path.')
+        context.argument('schema', options_list=['--schema', '-s'],
+                         arg_type=get_three_state_flag(),
+                         help='Switch to skip the operation and retrieve the PnP repository '
+                         'interface entity schema. Default: false')
+
+    with self.argument_context('iot dt monitor-events') as context:
+        context.argument('repair', options_list=['--repair'],
+                         arg_type=get_three_state_flag(),
+                         help='Reinstall uamqp dependency compatible with extension version.')
+        context.argument('consumer_group', options_list=['--consumer-group', '--cg'],
```

Comment:
`interface` should be moved here along with other relevant monitoring arguments. Also consider moving common monitoring arguments (if there are any) with single intention to the base `iot` command group.

Reply:
fixed as mentioned in the comment